test/test_linalg.py

```diff
@@ -5241,7 +5242,7 @@ def run_subtest(matrix_size, batches, device, pivot, singular=False, a=None):
                     self.assertEqual(np.matmul(P_nopiv_, np.matmul(L_nopiv_, U_nopiv_)), a)
 
                     k = min(rows, columns)
-                    self.assertEqual(nopiv, torch.arange(1, 1 + k, device=device, dtype=torch.int32).expand(a.shape[:-2] + (k, )))
+                    self.assertEqual(nopiv, torch.arange(1, 1 + k, device=device, dtype=nopiv.dtype).expand(a.shape[:-2] + (k, )))
```

Comment:
This change is not necessary, as we don't have int64 pivots at the moment, right?